docs/source/blog/arc-collaboration.md

```diff
@@ -0,0 +1,90 @@
+---
+blogpost: true
+date: Feb 28, 2025
+author: Will Graham
+location: London, England
+category: update
+language: English
+---
+
+# Spatial Navigation Feature Update
+
+_This is a short summary of the spatial navigation features introduced during Jan-Feb 2025._
+
+This work was part of a collaboration involving Sepiedeh Keshavarzi (Cambridge University), ARC (UCL), and the core `movement` development team and has been funded by the Wellcome Trust.
```

Comment:
I would spell out "ARC". Most visitors to thir website won't know what that means.